test-data/unit/fixtures/object_with_init_subclass.pyi

```diff
@@ -0,0 +1,53 @@
+from typing import Sequence, Iterator, TypeVar, Mapping, Iterable, Optional, Union, overload, Tuple, Generic
+
+
+class object:
+    def __init__(self) -> None: pass
+    def __init_subclass__(cls, **kwargs) -> None: pass
```

Comment:
This doesn't match typeshed, see https://github.com/python/typeshed/blob/master/stdlib/2and3/builtins.pyi. I would prefer to keep such subtle things in fixtures closer to the truth.